5.1.thats_the_way.py

```diff
@@ -0,0 +1,21 @@
+"""
+Writer: Ranel Ben Simman Tov, 315967703
+"""
+
+import os
+
+
+def find_files_with_prefix(path, prefix='deep'):
```

Comment:
Please add a main function that uses this function, and proves your code is working.
add the files directory to the submission aswell

Reply:
done